pyproject.toml

```diff
@@ -113,6 +113,10 @@ source-includes = [
 requires = ["pdm-backend"]
 build-backend = "pdm.backend"
 
+[tool.ruff]
+
+# Support python 3.8
+target-version = "py38"
```

Comment:
Did you run into a problem without this directive? I only ask because my understanding is that this [isn't necessary](https://docs.astral.sh/ruff/settings/#target-version) when `project.requires-python` is defined in [`pyproject.toml`](https://github.com/getpelican/pelican/blob/8fd94ea025cc0c2f29bf11dd63f02fd04e5add5c/pyproject.toml#L30).

Reply:
This was just me thrashing around when I had troubles with python 3.8 type hints. I thought it might help, but I think it didn't.

Removed.